lib/forklift/box_loader.rb

```diff
@@ -63,13 +63,15 @@ def process_versions(config, versions)
           katello_box = Marshal.load(
             Marshal.dump(config['boxes']["#{os}-katello-nightly"])
           )
-          katello_box['options'] ||= []
-          katello_box['options'] << " --version #{version}"
+          katello_box['ansible'] ||= {}
+          katello_box['ansible']['variables'] = { 'foreman_version' => version }
 
           capsule_box = Marshal.load(
             Marshal.dump(config['boxes']["#{os}-capsule-nightly"])
           )
+          capsule_box['ansible'] ||= {}
           capsule_box['ansible']['server'] = "#{os}-katello-#{katello_version}"
+          capsule_box['ansible']['variables'] = { 'foreman_version' => version }
```

Comment:
I would be ok with pulling out the dynamic boxes, I think if a user wants a particular box they could just create it with the right variables. Having fewer boxes listed by default in forklift would be better IMHO, especially if the boxes.yaml.example includes an example of a particular versioned box